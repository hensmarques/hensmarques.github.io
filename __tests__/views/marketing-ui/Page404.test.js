import React from 'react';
import Page404 from '../../../src/views/marketing-ui/Page404';

describe('<Page404/>', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<Page404 />);
    expect(wrapper).toMatchSnapshot();
  });
});
