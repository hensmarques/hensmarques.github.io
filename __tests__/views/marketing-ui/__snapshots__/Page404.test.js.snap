// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Page404/> renders and matches the snapshot 1`] = `
<div
  id="page404"
>
  <Connect(Nav)
    className="page404-header"
  />
  <div
    className="content"
  >
    <h2
      className="page-header"
    >
      Sorry, page you are looking for not found.
    </h2>
    <h1
      className="page-header-main"
    >
      404
    </h1>
  </div>
  <Footer
    className="page404-footer"
  />
</div>
`;
